doc/source/index.rst

```diff
@@ -58,7 +58,7 @@ basic statistical operations, random simulation and much more.
     ^^^^^^^^^^
```

Comment:
The `^` bit looks misaligned.

Reply:
Curiously, it look good locally. Soft tab maybe?

The "Contributor's Guide" section title also looks incorrectly marked up with `^`.